content/Payment Platforms options - Gonzalo Mercadante.md

```diff
@@ -0,0 +1,92 @@
+
+
+### Virtual goods
+
+There a lot of apps that use coins or credits to give the users access to certain functionalities, and for this case, you need to use In-app purchases. Despite the fact that most of the logic is managed in the frontend, because the payments are managed by apple/google in the device, the purchases should be validated also in the server.
+As with any feature that has anything to do with money, security is very important when implementing in-app purchases. Specifically, it’d be good to ensure that when you’re providing a user with a paid resource, they have actually paid for it.
+So, after the purchases are made, the frontend should send the receipt data to the server (encrypted data) and then the server should validate the receipt with apple/google servers.
+
+### Physical goods
+
+For the apps that sell physical goods, there are more options to use. Some of them: Stripe, Braintree, Dwolla, Paypal.
+Let’s take a look at each one.
+
+
+### Stripe
+
+Founded in 2011, Stripe provides technical infrastructure to operate online payment systems. It facilitates both private persons and business to accept online payments.
+It’s one of the best payment options, easy to implement and with good documentation.
+Accepts currencies from over 139 countries, and it automatically converts foreign currency into your account currency. It is available for over 30 countries, but accepts payments for almost all countries.
+The fee is 2.9% + 0.30$ per transaction, and supports subscriptions and discounts. Also,
+provides volume discounts for big companies, with more than $80.000 a month in sales.
+Credit cards: Visa, MasterCard, American Express, JCB, Discover, Diners Club, UnionPay. Accept international cards for an additional 1%. If the payment currency differs from your payout currency, a 1% conversion fee will also apply
+
+Some companies using stripe: Slack, Booking, Shoppify, Xero, National Geographic, etc.
+
+
+### Braintree
+
+Founded in 2007 and acquired by PayPal on Sep 26, 2013.
+Similar to Stripe, they have a good documentation, and let you take payments in over 130 currencies, but charges an additional 1% fee for international payments.
+The fee is 2.9% + 0.30$ per transaction and supports subscriptions and discounts. Provides volume discounts for big companies.
+Credit cards: Visa, MasterCard, American Express, JCB, Discover, Diners Club, UnionPay.
+Braintree allows you to accept Venmo and Paypal, so if you need to use Paypal in your app, then Braintree is better than Stripe. The fee for Paypal is charged directly from Paypal but it’s the same as Braintree.
+
+Some companies using Braintree: Airbnb, Uber, Github, Dropbox, Skyscanner, etc.
+
+
+### Dwolla
+
+Dwolla is an online payment solution specifically designed for better bank transactions, customer management, and account verification. It doesn’t include payments with credit cards.
```

Comment:
WDYT about changing the last sentence to. `It doesn't offer credit card payments support` or something similar.